File: dart-language/charge-system-demo/client-ng-dart/lib/src/history/history_item_component.dart
import 'package:angular/angular.dart';

@Component(
  selector: 'my-history-item',
  template: '''
   <div class="history-item">
   <div class="time">Time: {{time}}</div>
   <div class="value">Amount: \$ {{value}}</div>
   </div>
  ''',
  styles: ['''
  .history-item { margin: 10px 20px; padding: 10px; border-radius: 10px; background-color: #999; color:  #fff; line-height: 30px; }
  '''],
  directives: [coreDirectives]
)
class HistoryItemComponent {
  @Input()
  String time;

  @Input()
  num value;
}

File: dart-language/charge-system-demo/client-ng-ts/projects/at-antd/src/app/history/history-item.component.ts
import { Component, Input } from '@angular/core';

@Component({
  selector: 'ant-history-item',
  templateUrl: './history-item.component.html',
  styleUrls: ['./history-item.component.less']
})
export class HistoryItemComponent {
  @Input()
  time: string;

  @Input()
  value: number;
}
